code/obj/flock/structure/collector.dm

```diff
@@ -1,17 +1,28 @@
 //
-//Collector structure
+// # Collector structure
 //
 /obj/flock_structure/collector
 	name = "Some weird lookin' pulsing thing"
 	desc = "Seems to be pulsing."
 	flock_id = "Collector"
-	var/drawfromgrid = 0 //does it draw from the local apc if its strong enough.
-	var/active = 0 //is it active?
-	var/maxrange = 5 //max range for the thing.
-	var/connected = 0 //amount of tiles "connected" to.
-	var/list/connectedto = list() //the tiles its connected to
+	health = 60
+	/// does it draw from the local apc if its strong enough.
+	var/drawfromgrid = 0
+	/// is it active?
+	var/active = 0
+	/// max range for the thing.
+	var/maxrange = 5
+	/// amount of tiles "connected" to.
+	var/connected = 0
+	/// the tiles its connected to
+	var/list/connectedto = list()
```

Comment:
why do you need this first var... length of list

pretty sure the list can be typed and therefore used with as()

Reply:
idk i prolly thought that recalculating length each and every time the wouldbevar is accessed would be inperformant or something. its been months since i wrote that code